stackgl_modules/package.json

```diff
@@ -13,21 +13,22 @@
     "box-intersect": "plotly/box-intersect#v1.1.0",
     "convex-hull": "^1.0.3",
     "delaunay-triangulate": "^1.1.6",
-    "gl-axes3d": "^1.7.0",
-    "gl-cone3d": "^1.6.0",
-    "gl-error3d": "^1.0.16",
+    "gl-axes3d": "github:hborchardt/gl-axes3d#83caafc058b2d60b6006e26432967c411c125fe1",
```

Comment:
Please paste a link to the `gl-axes3d` pull request here.

Reply:
https://github.com/gl-vis/gl-axes3d/pull/23